lib/form.js

```diff
@@ -21,7 +22,7 @@ var qs = require('qs');
 
 module.exports = function(req, opts){
   req = req.req || req;
-  opts = opts || {};
+  opts = opts && clone(opts) || {};
```

Comment:
Yep, it's broken as is though. An alternative fix would be to do something in _koa-bodyparser_. That would only solve it there though, and would probably end up just being the same clone anyway. https://github.com/koajs/bodyparser/blob/master/index.js#L85

Any other suggestions? This shouldn't really be a complex object, but it's hard to say what people are actually passing in. clone has a "trust me there are no circuits" option, but I think using that would be a semver-major (should someone be passing such an object in as options today). We could also manually copy the options off, but we'd need to update the available options for _raw-body_ etc as those are added.